lib/buildkite/builder/github.rb

```diff
@@ -11,7 +11,7 @@ class Github
       ACCEPT_HEADER = 'application/vnd.github.v3+json'
       LINK_HEADER = 'link'
       NEXT_LINK_REGEX = /<(?<uri>.+)>; rel="next"/.freeze
-      REPO_REGEX = /github\.com(?::|\/)(.*)\.git\z/.freeze
+      REPO_REGEX = /github\.com(?::|\/)(.*?)(?:\.git)?\z/.freeze
```

Comment:
Hm, I'm not sure I understand why `(.*)` became optional: `(.*?)`

Reply:
The `?` allows for the `.*` to not greedily pick up the `.git` extension.

Without: https://rubular.com/r/bMSZ2cUtIw9zze
With: https://rubular.com/r/btm5lOuaB29efN

Reply:
Ah my bad. It's `(.*?)` and not `(.*)?`. Too early on a Monday morning 😄